code/model/editableformfields/EditableCheckbox.php

```diff
@@ -61,4 +63,8 @@ public function migrateSettings($data)
 
         parent::migrateSettings($data);
     }
+
+	public function isCheckBoxField(){
```

Comment:
Actually, can you just test `$field instanceof EditableCheckbox` or make this a more generic method in the base class? If I'm looping a list of UDF field objects I can't check `$field->isCheckBoxField()` because it won't exist on non-checkbox field classes